iana/entid.go

```diff
@@ -0,0 +1,20 @@
+package iana
+
+// EntID represents the Enterprise IDs as set by IANA
+// https://www.iana.org/assignments/enterprise-numbers/enterprise-numbers
+type EntID int
+
+// EntIDCiscoSystems is IANA Enterprise ID for Cisco Systems
+const EntIDCiscoSystems EntID = 9
```

Comment:
NIT: I think that if you have a const scope like the following, you don't need to have a comment for each of the elements:

```
// See IANA for values.
const (
	EntIDCiscoSystems EntID = 9
)
```

This will leave it in better shape for extending it with other values.